docs/Manual/05 Targets.md

```diff
@@ -2,7 +2,7 @@
 [//]: # (Version: v1.2.1)
 
 # Section 5: Targets
-Accera is a cross compiler, which means that it can generate code for different target platforms. A target is described using the `Target` class. Accera already knows about many different targets, for example:
+Accera is a cross compiler since it can generate executable code for different target platforms that can be described using the `Target` class. Accera already supports many different targets, for example: 
```

Comment:
"A target is described using the `Target` class." is a shorter and clearer sentence here.